Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM golang:1.20.2 as builder
+FROM --platform=$TARGETPLATFORM docker.io/library/golang:1.20.4 as builder
+
+ARG TARGETPLATFORM
+ARG BUILDPLATFORM
```

Comment:
what is the purpose of those?

Reply:
> what is the purpose of those?

none, leftovers from a previous iteration